microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,89 +3,152 @@ description: Version Models used for Microsoft Edge WebView2
+
+To manually look up the Runtime or browser version requirements for full API support for a specific SDK version, navigate to [Release notes for WebView2 SDK](../release-notes.md).  Then check the section for the specific SDK release.
+
+In your WebView2 app, to programmatically test whether a recent API is supported, a best practice is to check for support by using `QueryInterface` or `try/catch`.  This approach is described below.
+
+
+## Testing against Microsoft Edge preview channels
+   
 > [!IMPORTANT]
-> When developing [Evergreen WebView2 apps][Webview2ConceptsDistributionEvergreenDistributionMode], regularly test your app against the latest versions of the WebView2 Runtime and non-stable Microsoft Edge channels.  Because the web platform is constantly evolving, regular testing is the best way to ensure your app performs as intended.  
+> When developing Evergreen WebView2 apps, regularly test your app against the latest Microsoft Edge preview channels and the WebView2 Runtime.  Because the web platform is constantly evolving, regular testing is the best way to ensure your app performs as intended.
+
+For information about Evergreen WebView2 Runtime distribution, navigate to [Distribute a WebView2 app and the WebView2 Runtime][Webview2ConceptsDistribution].
+
 
-### Determine WebView2 Runtime requirement  
+## Testing whether the installed Runtime supports a recent API
 
-Depending on which SDK you use, consider the following items:  
+When you develop a WebView2 app using a recent version of the WebView2 SDK, when you use a recently added API, it's a good practice to test the client's installed WebView2 Runtime to check whether that API is present.  How your app programmatically tests for API support depends on the coding platform.
 
-*   **Win32 C/C++**.  When using `QueryInterface` to obtain a new interface, verify a return value of `E_NOINTERFACE`.  The value may indicate that the WebView2 Runtime is a previous version, and doesn't support that interface.  For an example of how it works, navigate to [Line 622 - AppWindow.cpp][GithubMicrosoftedgeWebview2samplesSampleappsWebview2apisampleAppwindowCppL622].  
-*   **.NET and WinUI**.  Check for a `No such interface supported` exception when using methods, properties, and events that were added to more recent SDKs.  The exception may occur when the WebView2 Runtime is a previous version, and does not support the APIs.  
+*   **Win32 C/C++**.  When using `QueryInterface` to obtain a new interface, test for a return value of `E_NOINTERFACE`.  That return value likely indicates that the client's WebView2 Runtime is an older version that doesn't support that interface.  For an example of checking for the existence of specific WebView2 APIs in the Runtime, navigate to [Line 622 - AppWindow.cpp][GithubMicrosoftedgeWebview2samplesSampleappsWebview2apisampleAppwindowCppL622].  This file wraps WebView2 API calls in the `CHECK_FAILURE` macro function, defined in `CheckFailure.h`.
```

Comment:
> Line 622 - AppWindow.cpp

Let's advise folks to search for `try_query` on AppWindow.cpp to look for examples instead of giving out a line number since that number will end up changing all the time as we update the sample.